src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -219,7 +219,8 @@ public StudentParticipation startExercise(Exercise exercise, Participant partici
             participation = startProgrammingExercise((ProgrammingExercise) exercise, (ProgrammingExerciseStudentParticipation) participation);
         }
         else {// for all other exercises: QuizExercise, ModelingExercise, TextExercise, FileUploadExercise
-            if (participation.getInitializationState() == null || participation.getInitializationState() == UNINITIALIZED || participation.getInitializationState() == FINISHED) {
+            if (participation.getInitializationState() == null || participation.getInitializationState() == UNINITIALIZED
```

Comment:
can you add a comment why you have changed this here?

Reply:
👍  cabd837